.github/workflows/post-merge-package-for-build.yml

```diff
@@ -43,6 +50,6 @@ jobs:
       - name: Deploy SAM app
         uses: alphagov/di-devplatform-upload-action@v3.2
```

Comment:
We can update this to the one login version now

Reply:
Good spot!

Reply:
Done ✅